src/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiProperty } from "@nestjs/swagger"
+import { IsEmail, IsNotEmpty, MinLength} from "class-validator"
+
+export class CreateUserDto {
+    @ApiProperty({
+        example: 'Christelle',
```

Comment:
Can you put an example of a full name instead to avoid confusing people who will use the api